src/app/app.component.ts

```diff
@@ -243,6 +239,15 @@ export class AppComponent {
     })).subscribe()
   }
 
+  updateTitle(event: RouterEvent) {
+      let event_url_array = event.url.split('/')
+      if (event_url_array && (event_url_array.length > 1)){
+        if (event_url_array[1] === 'home' || event_url_array[1] !== 'asset') {
+          this.titleService.setTitle(this.title)
+        }
+      }
```

Comment:
Isn't that equivalent to:

```javascript
      if (event_url_array && event_url_array.length > 1 &&
          (event_url_array[1] === 'home' || event_url_array[1] !== 'asset') {
        this.titleService.setTitle(this.title)
      }
```

I'm not sure it's worth making that change. Depends on what you find more readable.

Reply:
Yep, it was for readability!